version.js

```diff
@@ -57,65 +57,64 @@ const STACK_VERSION = {
 
 const ADDONS = {
 	'babel-polyfill': '^6.26.0',
-	'date-fns': '1.27.2',
 	'focus-outline-manager': '^1.0.2',
-	immutablediff: '0.4.4',
 	'normalize.css': '5.0.0',
-	'path-to-regexp': '2.0.0',
-	prettier: '1.6.1',
-	'redux-batched-subscribe': '0.1.6',
-	'redux-undo': 'beta',
-	'redux-saga': '0.15.4',
-	'react-addons-perf': '15.4.2',
-	'react-autowhatever': '10.1.0',
-	'react-debounce-input': '3.1.0',
-	'react-immutable-proptypes': '2.1.0',
-	'react-jsonschema-form': '0.51.0',
-	'react-tap-event-plugin': '2.0.0',
-	'react-virtualized': '9.10.1',
-	slugify: '1.1.0',
-	'whatwg-fetch': '2.0.3',
+	'path-to-regexp': '^2.0.0',
+	'react-addons-perf': '^15.4.2',
+	'react-tap-event-plugin': '^2.0.0',
+	'whatwg-fetch': '^2.0.3',
 };
 
 const VERSIONS = Object.assign({}, ADDONS, {
-	// deps
+	// deps: js libs
```

Comment:
what is the meaning of that ? a dependency is a js lib aka package ...

Reply:
I'll rename it to `non component libs`